### ISSUE TITLE
[Project Addition] : Rice Image-Analysis Using DL

### ISSUE THREAD

**Issue by SayantikaLaskar** [abhisheks008/DL-Simplified#826]
Deep Learning Simplified Repository (Proposing new issue)
🔴 Project Title : Rice Image-Analysis Using DL

🔴 Aim :Implement different algorithms like CNN, VGG16, ResNet

🔴 Dataset : To address person detection on rice image classification, we can either propose CNN from scratch or leverage a transfer learning approach with pre-trained models like VGG16 or ResNet50. Our approach involves training on a curated dataset specific to this project, fine-tuning models for accurate detection. Additionally, integrating an alert mechanism ensures timely notifications to authorities upon detecting individuals on tracks.
 https://www.kaggle.com/datasets/muratkokludataset/rice-image-dataset

🔴 Approach : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

📍 Follow the Guidelines to Contribute in the Project :
You need to create a separate folder named as the Project Title.
Inside that folder, there will be four main components.
Images - To store the required images.
Dataset - To store the dataset or, information/source about the dataset.
Model - To store the machine learning model you've created using the dataset.
requirements.txt - This file will contain the required packages/libraries to run the project in other machines.
Inside the Model folder, the README.md file must be filled up properly, with proper visualizations and conclusions.
🔴🟡 Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
"Issue Title" and "PR Title should be the same. Include issue number along with it.
Follow Contributing Guidelines & Code of Conduct before start Contributing.
✅ To be Mentioned while taking the issue :

Full name : Sayantika Laskar
GitHub Profile Link : https://github.com/SayantikaLaskar
Email ID : sayantikaLaskar2002@gmail.com
Approach for this Project : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.
What is your participant role? (Mention the Open Source program)
GSSoC'24 participant
Happy Contributing 🚀

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Assigned @SayantikaLaskar
